src/elements/ix-video.ts

```diff
@@ -1,10 +1,10 @@
 import {html, LitElement, PropertyValues} from 'lit';
 import {customElement, property, state} from 'lit/decorators.js';
 import {createRef, ref} from 'lit/directives/ref.js';
-import videojs, {VideoJsPlayerOptions} from 'video.js';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+import {VideoJsPlayer, VideoJsPlayerOptions} from 'video.js';
```

Comment:
I must have missed it, but what happened that resulted in this import changing?

Reply:
`videojs` module is being dynamically imported below. These two just `type` imports. Should probably change this to be `import type {...}` 🤔 .

Reply:
Ah gotcha, that makes sense 👍

Reply:
Yeah that might make it a little clearer imo